### PR TITLE
PAYARA-1039

### PR DESCRIPTION
**Pull request by mulderbaba**
We need a public api jar(s) on maven central so people can use public Payara apis.

This PR should be tested after getting artefacts built from https://github.com/payara/Payara-API repository.

### CODE REVIEW

File: appserver/payara-appserver-modules/payara-jsr107/pom.xml [hunk 1]
```diff
@@ -28,13 +28,16 @@
...
         <dependency>
             <groupId>javax</groupId>
             <artifactId>javaee-api</artifactId>
             <version>7.0</version>
-            <type>jar</type>
+        </dependency>
+        <dependency>
+            <groupId>fish.payara</groupId>
+            <artifactId>payara-api</artifactId>
+            <version>4.1.1.171-SNAPSHOT</version>
```
Comment (Pandrex247) [inline appserver/payara-appserver-modules/payara-jsr107/pom.xml]: Could you change this to `${project.version}` please? It's nice to keep consistency, and makes keeping versions in line easier

**Comment by michaelranaldo** [comment]
Jenkins test please

**Comment by michaelranaldo** [comment]
CI, please ignore

**Comment by smillidge** [comment]
jenkins test please

**Comment by smillidge** [comment]
jenkins test please

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by mulderbaba** [comment]
payaratests / JSR 107 (JCache) project should be updated with payara-api dependency definition.

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by mulderbaba** [comment]
jenkins test please

**Comment by lprimak** [comment]
jenkins test please

**Comment by mulderbaba** [comment]
jenkins test please

**Pull request closed without merging**